app.js

```diff
@@ -1,9 +1,27 @@
 const express = require("express");
 const { getTopics } = require("./controllers/topics.controllers");
+const { getApi } = require("./controllers/api.controllers");
+const { getArticleById } = require("./controllers/articles.controllers");
+const { getArticles } = require("./controllers/articles.controllers");
 const app = express();
 
 app.use(express.json());
 
+app.get("/api", getApi);
+
 app.get("/api/topics", getTopics);
 
+app.get("/api/articles", getArticles);
+
+app.get("/api/articles/:article_id", getArticleById);
+
+//Refactor of err truthy, as it can handle any error
+app.use((err, req, res, next) => {
+  if (err.status) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    res.status(500).send({ msg: "Internal Server Error" });
+  }
```

Comment:
I would recommend reading further in the error handling notes about making separate functions for error handlers as this application gets larger. Currently you aren't handling any PSQL errors which should have been tested for in your get/article/:article_id